README.md

```diff
@@ -13,13 +13,13 @@ Role Variables
 
 Available variables are listed below, along with default values (see `defaults/main.yml`):
 
-    prefix: test- # This should be overridden in inventory
-    domain_name: example.com # This also should be overridden in inventory
-    
+    prefix: test- # should be overriden
+    domain_name: example.com # used to create server_fqdn and meta-data, should be overridden
+    server_name: bastion
     server_subnet: "default-subnet" # should be on pair with default value from network_infra
     server_net: "{{ (prefix + 'otc-net') }}" # should be on pair with default value from network_infra
     security_group: "{{ (prefix + 'bastion_sg') }}"
```

Comment:
Please also add comment, that a security group with this name would be created (with SSH and ping ports) if not exist

Reply:
changes are added, pls review / confirm